spec/spec_helper.rb

```diff
@@ -42,6 +42,11 @@
     Rails.application.load_seed
   end
 
+  config.before :all do
+    # This is required to clean up after multiple `build(:document_type)` calls
```

Comment:
Minor: I think this is implicit, noting the other `clear` methods in here!

Reply:
Removed!